src/main/java/xyz/vegaone/easytrackingv3/domain/ProjectEntity.java

```diff
@@ -0,0 +1,38 @@
+package xyz.vegaone.easytrackingv3.domain;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+import javax.persistence.*;
+import java.util.List;
+
+@Getter
+@Setter
+@ToString
+@Entity
+@Table(name = "project")
+public class ProjectEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "project_name")
+    private String projectName;
```

Comment:
Just `name` should be enough since it's in a Project class. Don't forget about the changesets. 
The same for description.

The rest looks good.

Reply:
Done

Reply:
Could you please also update the changesets?